CONTRIBUTION.md

```diff
@@ -42,14 +42,10 @@ Unless your PR is directly related to code cleaning then your PR should not cont
 * Method/Class annotations should gain their own lines in front of the definition.
 * Parameter Annotations should be surrounded by exactly one space.
 * Comments should always start with a single space `// Comment`.
-* Import statments should be ordered in sections with exactly 1 empty line between them. Regular imports go in the front and static imports follow them.
-  * If you are using IntelliJ, this is the default order.
-  * If you are using Eclipse, you need to reorder them under `Preferences -> Java -> Code Style -> Organize Imports`.
-    1. `com.`, `.net`, and other imports, should be in alphebetical order
-    2. (empty line)
-    3. `java.`, `javax.`, etc.
-    4. (empty line)
-    5. All other static imports
+* Import statments should be ordered in sections with exactly 1 empty line between them.
```

Comment:
Still a hard no from me

Reply:
It would be nice if everyone could use the `organize imports` feature and get as close to the same result as possible. I usually end up avoiding doing so (instead, handling them more manually), due to the massive ordering changes of preexisting imports that it usually results in.

Reply:
Sure it would be nice but if both ide's do different things then it's just not ideal to require the develop to handle it manually.

Reply:
I handle it manually precisely because there is no standard, and I want to avoid making unnecessary changes to lines of code I'm not actually intending to change. It's not having a standard that I can at least potentially configure my IDE to follow that causes me to do things manually.

Reply:
I would like an agreed upon standard precisely so that I can then configure my IDE to match that standard as closely as possible, and stop having to do things manually most of the time.

Reply:
The idea behind the standard was to have our configs follow it but you can't change one the configs to make them follow the standard set out.

Reply:
The idea behind a contribution guide is to establish standards in general. It not at all out of place to at least establish some such standards for the organization of imports, since that is a thing that an IDE can be configured to do. It does belong somewhere in the guide.

Reply:
And I don't really care what the standards for imports are; I just want to know _what_ they are, so that I can stop having to organize imports so manually most of the time.

Reply:
I'll relent, I'm not super happy as the standard that has been put in the guide falls short and puts a lot of work on the developer if they're ide doesn't support the required standard but I'll agree that it's better than nothing. So I'm signing off. Just your review to be finalized now :)

Reply:
It doesn't have to be mandatory, but it should exist for anyone that wants to make the `organize imports` feature usable without imposing massive oscillating changes to the codebase. I can't stress enough how **not having** such a standard has made extra work for me; not the other way around.

And given that such standards have not existed, if I had been using `organize imports` regularly this whole time, then I would have been constantly changing virtually all of the imports in every file I touched, and then others would have been constantly reverting them back in every file they touched. That's clearly something that should be avoided, and it's taken *"alot of work"* to do so, mostly as a result of not having standards.

Reply:
Then let's set a standard now and Modify it as we go if we find it doesn't work out or causes more work. I've signed off so it's just your rejections that need to be sorted out :)